ESET/ESETEndpointAntiVirus.pkg.recipe

```diff
@@ -23,9 +32,9 @@
                 <key>source_path</key>
                 <string>%RECIPE_CACHE_DIR%/downloads/%NAME%.dmg/Resources/Installer.pkg</string>
                 <key>destination_path</key>
-                <string>%RECIPE_CACHE_DIR%/%NAME%.pkg</string>
+                <string>%RECIPE_CACHE_DIR%/%NAME%-%version%.pkg</string>
```

Comment:
I believe you will need to also change this in the Munki recipe since it is looking for the package solely based on the `%NAME%` var: https://github.com/autopkg/jbaker10-recipes/blob/master/ESET/ESETEndpointAntiVirus.munki.recipe#L45 and https://github.com/autopkg/jbaker10-recipes/blob/master/ESET/ESETEndpointAntiVirus.munki.recipe#L16

Reply:
Taken care of in https://github.com/autopkg/jbaker10-recipes/pull/11